python/hail/docs/other_resources.rst

```diff
@@ -0,0 +1,32 @@
+.. _sec-other_resources:
+
```

Comment:
Either add the SQL section to this page or move the hadoop glob patterns to a separate page. Then this page would look like:

```
.. toctree::
    :maxdepth: 2
 
    Hadoop Glob Patterns <hadoop_glob_patterns>
    SQL <sql>
```